specs/mediawiki/v1/mobileapps.yaml

```diff
@@ -47,8 +47,6 @@ paths:
             params:
               domain: en.wikipedia.org
               title: Main_Page
-            headers:
-              cache-control: no-cache
           response:
```

Comment:
Now the next monitoring test is identical to this one, so the whole x-ample can be removed.